auth_helper.py

```diff
@@ -0,0 +1,68 @@
+import os
+import pathlib
+
+import json5
```

Comment:
The inbuild `json` library should be enough right?

Reply:
Yep this is my bad this Auth helper code is borrowed from something else I wrote that did require json5. Will address.